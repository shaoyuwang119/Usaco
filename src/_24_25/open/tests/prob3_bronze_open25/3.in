50 50
yyyyyhwskoyuepyqaigcxofbrmtscfmbfrnblbgcdofebbhacp
1 49
1 50
1 19
5 16
4 49
11 27
1 47
9 49
6 48
1 48
2 16
1 43
8 48
2 49
1 49
3 49
3 49
1 19
4 50
3 49
1 17
10 49
4 16
7 48
9 49
25 28
2 50
1 50
2 43
1 50
1 18
1 50
3 48
2 46
3 41
47 49
1 50
43 45
16 47
18 22
3 18
1 45
1 50
4 49
31 33
3 48
4 48
9 47
3 20
3 18
