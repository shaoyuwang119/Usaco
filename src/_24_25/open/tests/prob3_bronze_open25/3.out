560
560
20
20
500
-1
506
400
396
506
20
440
352
540
560
520
520
20
500
520
20
380
20
374
400
-1
540
560
420
560
20
560
462
484
340
-1
560
-1
224
-1
20
483
560
500
-1
462
440
330
20
20
